bekeplar/EPIC-Mail
# Ft fetch user signup 165015198

What this PR does?
powers up front end feature user able to signup for an account
Tasks completed
-Writing JS functions to enable user login
- client side validations

How should this be manually tested?
On this Pull- request navigate to the files changed, tab to see the various files changed

Pivotal stories attached to the PR
[#165015198, user able to signup]

kindly review and give me feedback on necessary improvements
@Emmanuelaaron, @alexxsanya, @kyakusahmed, @neelxie @nanfuka @Wabuluka
